cli/tests/pcluster/test_commands.py

```diff
@@ -160,3 +161,28 @@ def test_update_failure_on_different_cluster_models(
         assert_that(caplog.text).contains(expected_message)
     else:
         assert_that(caplog.text).is_empty()
+
+
+@pytest.mark.parametrize(
+    "cluster_name, should_trigger_error",
```

Comment:
minor: should_trigger_error --> expected_error (to be aligned with other unit tests)

Reply:
I could do it, the only problem is that the expected_errors variables are usually not used for booleans but for strings containing the error. I could put the error instead of True and None instead of the error also, this would be strange as the error is the same every time but could be nice also.

Reply:
Spoke with Francesco, we leave as is now